README.md

```diff
@@ -1,30 +1,20 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Microverse!
 
-> One paragraph statement about the project.
-
-![screenshot](./app_screenshot.png)
-
-Additional description about the project and its features.
+> First project setting up a "Hello world" repository
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTLM
+- CSS
 
 ## Live Demo
 
 [Live Demo Link](https://livedemo.com)
 
-
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
```

Comment:
- [ ] Please add more information about how to setup the project 👍 It's always nice to let people who view your project know how to set it up and install the packages . also, you can remove the unnecessary headers that don't have extra information if you don't need them for your project.   
something as simple as this would be nice

```suggestion
## Setup

To get a local copy up and running follow these simple example steps.

`git clone https://github.com/TimmyChan99/Hello-world`
```